### PR TITLE
Update README.md

### LINKED ISSUE #62: Update README

### PR DESCRIPTION
**Pull request by keithradford**
closes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,47 @@
-# Getting Started with Create React App
+# clockwork
+[![Contributors][contributors-shield]][contributors-link]
+[![Stargazers][stars-shield]][stars-link]
+[![Issues][issues-shield]][issues-link]
+[![Website][website-shield]][website-link]
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+clockwork is a website built to simplify the experience of searching courses and building timetables for the [University of Victoria](https://uvic.ca) (UVic). clockwork is a TypeScript app that uses React on the front-end and Firebase on the back-end.
```
Comment (zthh626) [inline README.md:7]: Clockwork in both sentences is lowercase, is this intentional?

File: README.md [hunk 2]
```diff
@@ -1,59 +1,47 @@
...
 
-## Available Scripts
+## Contribute
+1. Clone the repo:
```
Comment (zthh626) [inline README.md:10]: We also need to downgrade to npm v6 until the fix for Storybook comes out

File: README.md [hunk 3]
```diff
@@ -1,59 +1,47 @@
...
-### `npm start`
+`npx jest --watch` will put Jest into watch mode, which will execute tests as files change.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Developer Tools
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+When developing in your local environment, to view components which require requests to populate with data you will need to enable CORS. Install the following extension to do so:
```
Comment (zthh626) [inline README.md:31]: Might be nice to add something about Safari:
https://stackoverflow.com/questions/4556429/disabling-same-origin-policy-in-safari
This is what I did

**Review by zthh626 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (keithradford) [reply README.md]: I thought it'd be cool aesthetics 😎  but I changed it

File: README.md [hunk 4]
```diff
@@ -1,59 +1,50 @@
-# Getting Started with Create React App
+# clockwork
+[![Contributors][contributors-shield]][contributors-link]
+[![Stargazers][stars-shield]][stars-link]
+[![Issues][issues-shield]][issues-link]
+[![Website][website-shield]][website-link]
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Clockwork is a website built to simplify the experience of searching courses and building timetables for the [University of Victoria](https://uvic.ca) (UVic). The website is a TypeScript app which uses React on the front-end and Firebase on the back-end.
```
Comment (zthh626) [inline README.md:7]: I'm fine with either aesthetic, you can change it back if you want. but other than that, lgtm.

**Review by zthh626 — APPROVED** [review]

**Pull request merged**